webapp/packages/core-ui/src/Tabs/Tab/baseTab.m.css

```diff
@@ -0,0 +1,106 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+.placeholder {
+  composes: placeholder from global;
+}
+
+.baseTab {
+  position: relative;
+  outline: none;
+  font-weight: normal;
+  cursor: pointer;
+  padding: 0;
+  white-space: nowrap;
+
+  height: 32px;
+  border-top: solid 2px transparent;
```

Comment:
please split this file to multiple files for each component
like Tab.m.css, TabIcon.m.css, TabTitle.m.css, TabList.m.css, TabPanel.m.css, TabMenu.m.css

Reply:
this is pretty dangerous from perspective of current refactor. this will cause for sure a lot of layout bugs. my proposal here is to keep this for a while. for me this construction seems pretty stable, cause for all of the tabs (or most of them) this construction did good job